solution/backend/regulations/templates/regulations/partials/recent_changes.html

```diff
@@ -2,17 +2,21 @@
 
 <div class="latest-changes">
 
-    {% comment %}<h3>Recent Documents for 42 CFR 430-460</h3>{% endcomment %}
+    <h3>Recent Rules</h3>
 
-    {% comment %}<supplemental-content class="related-rules" api_url="{% url 'regcore' %}" requested_categories="26"></supplemental-content>{% endcomment %}
+    <p class="recent-rules-descriptive-text">Includes 42 CFR 400, 430-460</p>
 
-    {% comment %}<hr class="latest-changes-horiz-rule"/>{% endcomment %}
+    <recent-changes-container api-url={% url "regcore" %} ></recent-changes-container>
+
+    <a href="{% url 'resources' %}?title=42&resourceCategory={{ fr_docs_category_name }}" class="action-btn default-btn link-btn">View More Changes</a>
```

Comment:
Tiny nit pic but can we keep the quotations consistant? I see {% url 'resources' %} in this line but line 9 its {% url "regcore" %} ?

Reply:
we need `'resources'` to be in single quotes because it is quoted within a double-quoted `href` attribute.